Assets/MixedRealityToolkit.Examples/Demos/HandTracking/Scenes/HandInteractionExamples.unity

```diff
@@ -337,7 +337,7 @@ PrefabInstance:
     - target: {fileID: 7307063430618434848, guid: 1093263a89abe47499cccf7dcb08effb,
         type: 3}
       propertyPath: sliderEndDistance
-      value: 0.06702003
+      value: 0.06702006
```

Comment:
This is a frequently used scene, it would be good to make sure @cre8ivepark  is aware of changes here. Could you please describe the changes you made in this scene?

Reply:
Not sure about this one. I didn't change the end sliderEndDistance in the editor.
I guess it happened through the script as sliderEndDistance  is being set in PinchSlider.cs as well : 
sliderEndDistance = Vector3.Dot(transform.InverseTransformPoint(value), GetSliderAxis());